dev/io.openliberty.microprofile.faulttolerance.4.0.internal_fat_tck/bnd.bnd

```diff
@@ -23,7 +23,7 @@ tested.features:\
   expressionlanguage-5.0,\
   mpmetrics-5.0,\
   jsonp-2.1,\
-  servlet-6.0
+  restfulws-3.1
```

Comment:
you should be able to remove appsecurity-5.0, cdi-4.0 and expressionlanguage-5.0 with the update to the server.xml.

Reply:
Done, thank you.